board-responsibilities.md

```diff
@@ -0,0 +1,23 @@
+## Responsibilities of Board of Directors and Technical Steering Committee
+
+## Board of Directors - Administrative or Executive Branch
+
+1. Define the strategic direction
+2. Final approval of - Budgets/Quarterly Reports
+   a. Fiscal oversight and accountability
+3. Legal issues
+4. Fundraising
+   a. Help develop and participate in a fundraising plan
+5. Risk Assessment
+6. Formation and oversight for relevant committees, including CoC working group and Election committee
+
+_Meetings held **quarterly** via Zoom. Directors not holding an office should plan on 5 hrs/month._
```

Comment:
Because the caption function of Google Meet is more powerful than Zoom, using it is more friendly to the hearing impaired and non-native English speakers.
```suggestion
_Meetings held **quarterly** via Google Meet. Directors not holding an office should plan on 5 hrs/month._
```

Reply:
I think the group members will likely have platform preferences. I'll remove the mention of a platform from both sections. :)